### PR TITLE
Finish account activation

### PR DESCRIPTION
**Pull request by hoangnguyen293**
screenshot from 2018-07-13 13-45-27

### CODE REVIEW

**Comment by hoangnguyen293** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -28,12 +29,33 @@ def remember
...
+    return false if digest.nil?
+    BCrypt::Password.new(digest).is_password?(token)
   end
 
   def forget
     update_attribute(:remember_digest, nil)
   end
+
+  def activate
+    update_attribute(:activated,    true)
```
Comment (hhhhuy112) [inline app/models/user.rb]: 1 space after comma

File: app/views/user_mailer/account_activation.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<h1><%= t "account_activation.sample_app" %></h1>
+
+<p>Hi <%= @user.name %>,</p>
+
+<p>
+<%= t "account_activation.welcome" %>
+</p>
+
+<%= link_to "Activate", edit_account_activation_url(@user.activation_token,
```
Comment (hhhhuy112) [inline app/views/user_mailer/account_activation.html.erb]: use I18n

**Comment by hhhhuy112** [comment]
commented

**Comment by hoangnguyen293** [comment]
ready
Đã fix anh chị check lại giúp em với ạ :)

**Comment by hhhhuy112** [comment]
merrged
You can merge it

**Pull request merged**